src/plone/app/mosaic/utils.py

```diff
@@ -98,6 +98,17 @@ def getContentLayoutsForType(pt):
     for item in hidden[:]:
         # undocumented feature right now.
         # need to figure out how to integrate into UI yet
+        # For type "Homepage"
```

Comment:
Don't understand this... How is that hidden for the type "Homepage"? There is no other reference to "Homepage" here....